src/js/main.js

```diff
@@ -162,17 +160,6 @@ export function setupForm () {
           close();
         }
       };
-      if (submitButton && typeof ga !== "undefined") {
-        submitButton.addEventListener("click", function () {
-          ga(
-            "send",
-            "event",
-            "interactive-forms",
-            "submitted",
-            window.location.pathname
-          );
-        });
-      }
```

Comment:
I think this is fair enough

Reply:
yeah the submit button is no longer useful, because apparently we are removing all marketo classes